internal/api/dto/price.go

```diff
@@ -572,11 +596,53 @@ type UpdatePriceRequest struct {
 type PriceResponse struct {
 	*price.Price
 	Meter *MeterResponse `json:"meter,omitempty"`
+
+	// TODO: Remove this once we have a proper price entity type
+	PlanID string `json:"plan_id,omitempty"`
 }
 
 // ListPricesResponse represents the response for listing prices
 type ListPricesResponse = types.ListResponse[*PriceResponse]
 
+// CreateBulkPriceRequest represents the request to create multiple prices in bulk
+type CreateBulkPriceRequest struct {
+	Items []CreatePriceRequest `json:"items" validate:"required,min=1,max=100"`
+}
+
+// CreateBulkPriceResponse represents the response for bulk price creation
+type CreateBulkPriceResponse struct {
+	Items []*PriceResponse `json:"items"`
+}
+
+// Validate validates the bulk price creation request
+func (r *CreateBulkPriceRequest) Validate() error {
+	if len(r.Items) == 0 {
+		return ierr.NewError("at least one price is required").
+			WithHint("Please provide at least one price to create").
+			Mark(ierr.ErrValidation)
+	}
+
+	if len(r.Items) > 100 {
+		return ierr.NewError("too many prices in bulk request").
+			WithHint("Maximum 100 prices allowed per bulk request").
+			Mark(ierr.ErrValidation)
+	}
+
+	// Validate each individual price
+	for i, price := range r.Items {
```

Comment:
Similarly here as well we need to make sure plan id is set when validating for bulk create